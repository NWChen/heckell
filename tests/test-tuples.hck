let t = (3, "hello", 'x');
print("t = \(t)");
