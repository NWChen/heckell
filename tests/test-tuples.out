t = (3, hello, 'x')
